### PR TITLE
EXPDAT-012/001 Foodsharing API

### PR DESCRIPTION
**Pull request by steschi**

### CODE REVIEW

File: app/src/androidTest/java/at/sw21_tug/team_25/expirydates/FoodSharingApiClientTests.kt [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        val usedPath = request2!!.path.toString()
+        Assert.assertEquals("/api/foodSharePoints/nearby?lat=47.070713&lon=15.439504&distance=20", usedPath)
+
+        Assert.assertEquals(dummyCsrfTokenCookie, request2.getHeader("Cookie"))
+
+        server.shutdown()
+    }
+
+
+}
```
Comment (bengaal) [inline app/src/androidTest/java/at/sw21_tug/team_25/expirydates/FoodSharingApiClientTests.kt]: Looks good, just please add a newline to the EOF (see [Standard](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)).

File: app/src/main/java/at/sw21_tug/team_25/expirydates/utils/FoodSharingAPIClient.kt [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            val itemLon = item.getDouble("lon")
+            val point = FoodSharePoint(id, name, itemLat, itemLon)
+            result.add(point)
+        }
+        
+        return result
+    }
+
+
+}
```
Comment (bengaal) [inline app/src/main/java/at/sw21_tug/team_25/expirydates/utils/FoodSharingAPIClient.kt]: Please add a newline to the EOF.

**Review by bengaal — CHANGES_REQUESTED** [review]
Looks really good to me. All the tests run locally as well.

File: app/src/androidTest/java/at/sw21_tug/team_25/expirydates/FoodSharingApiClientTests.kt (hunk 1 above)
Comment (ettha) [reply app/src/androidTest/java/at/sw21_tug/team_25/expirydates/FoodSharingApiClientTests.kt]: EOf added

**Review by bengaal — APPROVED** [review]

**Pull request merged**